### ISSUE TITLE
Negative Loss

### ISSUE THREAD

**Issue by ashwath98** [sampathweb/insight-clothing-attributes#8]
Hey.
I'm trying to follow the steps in clothing-attributes-moduler.ipynb

I initially was using torch version 1.4 and ran into some index errors[1 line in classifiers.py] but fixed it and the model resulted in Loss: -0.0180 Acc: 0.5318
after 18 epochs.

torch also raised some user warnings like
UserWarning: The use of the transforms.Scale transform is deprecated, please use transforms.Resize instead
and
UserWarning: Implicit dimension choice for softmax has been deprecated. Change the call to include dim=X as an argument. return F.softmax(self.model(x))

So I decided to test on an older version of torch [torch=0.4.1 and torchvision=0.2.2]
but still the same issue persists
I got the same warnings 
and the model resulted in -0.0180 Acc: 0

What do you think could be the source of the issue? The negative loss value hints towards something to do with the loss function. So should I specify a dim value to the softmax layer?

Thanks :)
